src/garage/torch/modules/tanh_gaussian_mlp_module.py

```diff
@@ -0,0 +1,208 @@
+"""GaussianMLPModule."""
+import abc
+
+import torch
+from torch import nn
+
+from garage.torch.distributions import TanhNormal
+from garage.torch.modules.mlp_module import MLPModule
+from garage.torch.modules.multi_headed_mlp_module import MultiHeadedMLPModule
+
+
+class TanhGaussianMLPBaseModule(nn.Module):
+    """
+    GaussianMLPModel.
```

Comment:
Please make these comments more descriptive. The name of the class is not a sufficient summary line.